catalog/resource_metastore_assignment.go

```diff
@@ -76,7 +77,6 @@ func ResourceMetastoreAssignment() common.Resource {
 					return err
 				}
 				d.Set("metastore_id", ma.MetastoreId)
-				d.Set("default_catalog_name", ma.DefaultCatalogName)
```

Comment:
if we marked it as `computed` we can still set it to the answer from backend?

Reply:
yes - I've added it back